recipe/meta.yaml

```diff
@@ -1,46 +1,49 @@
 {% set name = "pysteps" %}
-{% set version = "1.13.0" %}
+{% set version = "1.14.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/a1/22/ddc583bbae5b4cdd34c7cb2e30e96f88df4ad9197065502ad4e330d8e4b1/pysteps-{{ version }}.tar.gz
-  sha256: a3dbd72a6fe368b42d7761efe4a3b658b12224ec51e2cf7276641d2fc43cf20d
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pysteps-{{ version }}.tar.gz
+  sha256: 3437fd995fbef42a5c5384845a8746dea8f15b12d6ea1540051a46710af3f1c2
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   skip: true  # [py<310 or py>312]
 
 requirements:
   build:
-    - {{ compiler('c') }}
+    - {{ compiler("c") }}
     - {{ stdlib("c") }}
   host:
     - python
-    - numpy >=1.17
```

Comment:
@dnerini I regenerated the recipe with https://github.com/conda/grayskull. Note that the dependencies changes may not reflect some non-pyhton deps. If you know what is needed here, please add/re-add accordingly. What is in there now is the exact deps described in the package metadata.

Reply:
thanks a lot @ocefpaf , this looks great! I will need to have a closer look at the changes in the dependencies